pylint/checkers/base/docstring_checker.py

```diff
@@ -98,8 +98,8 @@ class DocStringChecker(_BasicChecker):
                 "type": "int",
                 "metavar": "<int>",
                 "help": (
-                    "Minimum line length for functions/classes that"
-                    " require docstrings, shorter ones are exempt."
+                    "Minimum line length for functions/classes that "
```

Comment:
Like other said previously, please refrain from making such cosmetic changes.

Reply:
Got it. I just want it to be consistent with the nearby option. Already revert.